dvc/remote/s3.py

```diff
@@ -112,9 +118,11 @@ def get_head_object(s3, bucket, path, *args, **kwargs):
         return obj
 
     @classmethod
-    def _copy_multipart(cls, s3, from_info, to_info, size, n_parts):
+    def _copy_multipart(
+        cls, s3, from_info, to_info, size, n_parts, extra_args={}
+    ):
         mpu = s3.create_multipart_upload(
-            Bucket=to_info.bucket, Key=to_info.path
+            Bucket=to_info.bucket, Key=to_info.path, **extra_args
```

Comment:
Let's make this(`_copy_multipart`) a regular method (along with `_copy`) and use `self.extra_args` directly and not pass it through other functions.